scc/building-from-source.md

```diff
@@ -449,3 +453,75 @@ Instead of a `GitRepository` object, an `ImageRepository` is created:
 `ImageRepository` provides the same semantics as `GitRepository`,
 except that it looks for source code in container image registries rather than
 Git repositories.
+
+## <a id="maven-artifact"></a>Maven Artifact
+
+This approach is intended to be used to aid integration with existing CI/CI
+systems (e.g.: Jenkins) and can pull artifacts from existing Maven repositories,
+including Jfrog Artifactory.
+
+At this time, there are no dedicated fields in the `Workload` resource for
+specifying the Maven artifact configuration; you need to fill in the
+`name`/`value` pairs in the `params` structure.
+
+Example:
+
+```yaml
+apiVersion: carto.run/v1alpha1
+kind: Workload
+metadata:
+  name: my-workload
+  labels:
+    app.tanzu.vmware.com/workload-type: web
+spec:
+  params:
+  - name: maven
+    value:
+    - name: groupId
+      value: com.example
+    - name: artifactId
+      value: spring-boot-initial
+    - name: version
+      value: RELEASE   # latest 'RELEASE' or a specific version (e.g.: '1.2.2')
+    - name: type
+      value: jar
+    - name: classifier # optional
+      value: sources   # optional
+```
+
+The `tanzu` CLI does not yet support creating workloads for Maven artifacts.
```

Comment:
I _think_ the big problem would be the validations that are included in `apps-cli-plugin`, e.g., preventing you from creating a workload with _solely_ `--param-yaml` - I've just noticed [this commit](https://github.com/vmware-tanzu/apps-cli-plugin/commit/1800afff834450f1745234a268393fb2f7506ac6) adding some different logic for handling that, so, perhaps for 1.2.0 that'll have arrived? (speculation on my side, @scothis would know more?)

Reply:
As there is no build including this new functionality for the `apps-cli-plugin` it's pretty hard for me to test that it's going to work.  I'm going to have to leave this statement as it is until I can get my hands on a new build of the plugin.